blaze/json_dumps_trusted.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import absolute_import, division, print_function
+
+import types
+from collections import Callable
+
+from .dispatch import dispatch
+from functools import partial, wraps
+
+from .json_dumps import json_dumps
+
+
+json_dumps_trusted_ns = dict()
```

Comment:
instead of the loop below this might be more clear as `json_dumps_trusted_ns = jsun_dumps.funcs.copy()`